### PR TITLE
Сhanges in user schema

### PR DESCRIPTION
**Pull request by nikitius20**
I deleted locations field in get profile, because there is no locations field in user and i dont even know why it was there
photo5197225288373746267
photo5197225288373746269

### CODE REVIEW

**Review by Yojuks — APPROVED** [review]

**Review by vasylneshchadimov — APPROVED** [review]

File: backend/controllers/UserController.ts [hunk 1]
```diff
@@ -14,9 +14,11 @@ const UserController = {
...
         description: true,
         imageUrl: true,
-        locations: true,
         subscribers: true,
-        subscriptions: true
+        subscriptions: true,
+        favourite: true,
+        visited: true,
+        personalLocations: true
       });
```
Comment (6mahdapihka9) [inline backend/controllers/UserController.ts:22]: why did you rename field locations into personalLocations?

it's obvious that there are only user's locations 🤔

Comment (6mahdapihka9) [inline backend/controllers/UserController.ts:22]: have you changed existed data in the mongoDB, so everything works fine?
